peer/peer.go

```diff
@@ -205,6 +205,30 @@ type MessageListeners struct {
 	// OnInitState is invoked when a peer receives an initstate message.
 	OnInitState func(p *Peer, msg *wire.MsgInitState)
 
+	// OnMixPR is invoked when a peer receives a mixpairreq message.
+	OnMixPR func(p *Peer, msg *wire.MsgMixPairReq)
```

Comment:
These all should match the wire names since this is going to be a part of the public peer API and we can't change it after merge without a major module version bump.

aka `OnMixPairReq`, `OnMixKeyExchange`, `OnMixCiphertexts` etc.